projects/components/src/overlay/modal/modal-service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable, Injector } from '@angular/core';
+import { PopoverOptions, POPOVER_DATA } from '../../popover/popover';
+import { PopoverRef } from '../../popover/popover-ref';
+import { PopoverService } from '../../popover/popover.service';
+import { ModalRef } from './modal-ref';
+
+@Injectable()
+export class ModalService extends PopoverService {
```

Comment:
Similar to the ref, I think we should avoid extending popoverservice, and instead use it - similar to how the existing overlay service works.